src/scripts/main.js

```diff
@@ -1 +1,67 @@
 'use strict';
+
+function createMessage(message, messageClass) {
+  let newMess = message;
+
+  if (messageClass === 'error') {
+    newMess = 'First promise was rejected';
+  };
+
+  document.body.insertAdjacentHTML('beforeend', `
+    <div data-qa="notification" class="${messageClass}">
+      ${newMess}
+    </div>
+  `);
+}
```

Comment:
make this function flexible. It should work properly if we will add reject handlers for other promises

Reply:
Done